samples/nhibernate/custom-mappings/sample.md

```diff
@@ -1,38 +1,33 @@
 related:
  - persistence/nhibernate
 ---
 
 
-Sometimes the database schema that gets generated by `NServiceBus.NHibernate` for the saga data types is insufficient. For example, storing strings with a different string length, want so use complex types, need to store DateTime values with a high precision or maybe want to tweak the eager versus lazy loading to improve performance. A custom mapping will be required to achieve this, so that the generated default mapping will not be used.
+Sometimes the database schema that is generated by `NServiceBus.NHibernate` is insufficient for saga data types. For example, storing strings with a different string length, wanting to use complex types, needing to store DateTime values with a high precision, or wanting to tweak the eager versus lazy loading rules might be better options. A custom mapping will be required to achieve these changes.
 
-Custom mapping options:
+Custom mapping can be added by:
 
- * Create a `*.hbm.xml` file for the saga data class
- * Use Fluent NHibernate (popular separate fluent API)
- * Use NHibernate.Mapping.Attributes
+ * Creating a `*.hbm.xml` file for the saga data class
+ * Fluent NHibernate (popular separate fluent API)
+ * NHibernate.Mapping.Attributes
  * Loquacious Configuration (native fluent API)
 
-There are probably even other options, but these are the most frequently used.
+There are other options, but these are the most frequently used. The performance difference between these options can hae small variances, but those will occur during endpoint startup, not when the endpoint is processing messages.
 
-Which method to use depends on if mapping as xml, fluent code or code attributes. There can be small performance differences during startup but not when processing messages.
+NOTE: There is no requirement to create a custom mapping for all sagas if a custom mapping is needed by only one saga. The NHibernate configuration in each sample type demonstrate how both generated and custom mappings can work simultaneously.
 
-NOTE: It is not required to create a custom mapping for all sagas if only one saga needs one. The NHibernate configuration in each sample type demonstrate how both generated and custom mappings can work simultaneously.
-
-NOTE: When creating a custom mapping then mapping the **primary key** and **unique indexes** must be done as part of that mapping.
-
-When using custom mappings ensure unique indexes for all mapped saga properties are correctly mapped. Especially when using optimistic concurrency, a transaction isolation level different from serializable and allow concurrent processing of messages.
-
-Not adding a unique constraint can result in duplicate saga entities as the second insert will **not** fail when inserting the same value when multiple messages are processed concurrently referencing the same saga instance.
+NOTE: When creating a custom mapping, the mapping of the **primary key** and **unique indexes** must be done included.
```

Comment:
"done included"?

Reply:
would "dun included" be more Minnesotan?